charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -779,68 +783,75 @@ vmagent:
     #    hosts:
     #      - vmagent.domain.com
 
+defaultDatasources:
```

Comment:
Why not reuse the [grafana.datasources](https://github.com/grafana/helm-charts/blob/07419bd5220eef66e1c4906390cb27d153de0bb6/charts/grafana/values.yaml#L612), should be more easy to see and understand by user.
And I don't see the need for `perReplica` (the old `createVMReplicasDatasources`), as it seems to serve only for debug purposes and works only on vmsingle, which I doubt is used. We can remove the field and consolidate all datasources under `grafana.datasources` instead of creating our own.

Reply:
https://github.com/grafana/helm-charts/blob/07419bd5220eef66e1c4906390cb27d153de0bb6/charts/grafana/values.yaml#L858

Reply:
createVMReplicasDatasources -> perReplica is needed when vmsingle instance count is > 1

Reply:
>https://github.com/grafana/helm-charts/blob/07419bd5220eef66e1c4906390cb27d153de0bb6/charts/grafana/values.yaml#L858

ok, I don't see they have check or restriction in code though.

>createVMReplicasDatasources -> perReplica is needed when vmsingle instance count is > 1

For what? I can only see for debug, and it's pretty strange way to create datasources for that.

Reply:
> For what?
for those, who are using VMsingle in HA mode, the same setup as we're now suggesting for VLogs users till cluster version is not available
> ok, I don't see they have check or restriction in code though.
There were complaints from users regarding dashboards, so we already moved `node-exporter-full` dashboard to configmap